index.js

```diff
@@ -170,17 +208,16 @@ function cli(process_args) {
       dockerEvents: true,
       statsinterval: 30,
       add: [ 'host=' + os.hostname() ],
+      debug: !!process.env.INSIGHT_DOCKER_DEBUG,
       token: process.env.INSIGHT_TOKEN,
       logstoken: process.env.INSIGHT_LOGSTOKEN || process.env.INSIGHT_TOKEN,
       statstoken: process.env.INSIGHT_STATSTOKEN || process.env.INSIGHT_TOKEN,
       eventstoken: process.env.INSIGHT_EVENTSTOKEN || process.env.INSIGHT_TOKEN,
       server: '.data.logs.insight.rapid7.com',
-      port: unbound
+      port: undefined
     }
   });
 
-
-
   if (argv.help || !(argv.token || argv.logstoken || argv.statstoken || argv.eventstoken) || !(argv.region)) {
     console.log('Usage: r7insight_docker [-l LOGSTOKEN] [-k STATSTOKEN] [-e EVENTSTOKEN]\n' +
```

Comment:
maybe move all flags on newlines for readability?

Reply:
they're grouped together currently